mirdata/cante100.py

```diff
@@ -0,0 +1,491 @@
+        minutes.append(float(minute.text) * 60)
+    seconds = []
+    for second in root.iter('duration_s'):
+        seconds.append(float(second.text))
+    for i in np.arange(len(minutes)):
+        durations.append(minutes[i] + seconds[i])
+
+    metadata = dict()
+    metadata['data_home'] = data_home
+    j = 0
+    for i in indexes:
+        metadata[i] = {
+            'musicBrainzID': identifiers[j],
+            'artist': artists[j],
+            'title': titles[j],
+            'release': releases[j],
+            'duration': durations[j],
+        }
+        j += 1
+
+    return metadata
+
+
+DATA = utils.LargeData('cante100_index.json', _load_metadata)
+
+
+class Track(track.Track):
+    """cante100 track class
+
+    Args:
+        track_id (str): track id of the track
+        data_home (str): Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets/cante100`
+
+    Attributes:
+        track_id (str): track id
+        identifier (str): musicbrainz id of the track
+        artist (str): performing artists
+        title (str): title of the track song
+        release (str): release where the track can be found
+        duration (str): duration in seconds of the track
+    """
+
+    def __init__(self, track_id, data_home=None):
+        if track_id not in DATA.index:
+            raise ValueError('{} is not a valid track ID in Example'.format(track_id))
+
+        self.track_id = track_id
+
+        if data_home is None:
+            data_home = utils.get_default_dataset_path(DATASET_DIR)
+
+        self._data_home = data_home
+
+        self._track_paths = DATA.index[track_id]
+        self.audio_path = os.path.join(
+            self._data_home, self._track_paths['audio'][0]
+        )
+        self.spectrum_path = os.path.join(
+            self._data_home, self._track_paths['spectrum'][0]
+        )
+        self.f0_path = os.path.join(self._data_home, self._track_paths['f0'][0])
+        self.notes_path = os.path.join(self._data_home, self._track_paths['notes'][0])
+
+        metadata = DATA.metadata(data_home=data_home)
+        if metadata is not None and track_id in metadata:
+            self._track_metadata = metadata[track_id]
+        else:
+            self._track_metadata = {
+                'musicBrainzID': None,
+                'artist': None,
+                'title': None,
+                'release': None,
+                'duration': None,
+            }
+
+        self.identifier = self._track_metadata['musicBrainzID']
+        self.artist = self._track_metadata['artist']
+        self.title = self._track_metadata['title']
+        self.release = self._track_metadata['release']
+        self.duration = self._track_metadata['duration']
+
+    @property
+    def audio(self):
+        """(np.ndarray, float): audio signal, sample rate"""
+        return load_audio(self.audio_path)
+
+    @property
+    def spectrum(self):
```

Comment:
I think this is a 2d spectrogram: time, frequency. or even 3d if you consider stereo channels. maybe rename it everywhere in mirdata.
what do you think @rabitt @magdalenafuentes ? should we have something more standard just in case there is another future dataset with spectrogram?